ex03.ex

```diff
@@ -126,31 +79,22 @@ defmodule Ex03 do
   Each element of the tuple is set to :x or :o if the corresponding
   player has occupied the square, or its number otherwise.
 
-
        X | X |
        --+---+--
          | O |        => { :x, :x, 3,   4, :o, 6,   7, :o, :x }
        --+---+--
          | O | X
-
-  There are eight winning positions (three horizontal, three vertical,
-  and two diagonal). Write a function that returns :x if X has won, :o
-  if O has won, and false otherwise.
-
-      iex> Ex03.won { :x, :o, 3,   :x, 5, :o,   :x, 8, :o }
-      :x
-
-      iex> Ex03.won { :o, :x, 3,   :x, :o, 6,   :x, 5, :o }
-      :o
-
-      iex> Ex03.won { :o, :x, 3,   :x, :o, 6,   :x, :o, 9 }
-      false
-
-  Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
-
+  def won({x, x, x, _4, _5, _6, _7, _8, _9}), do: x
```

Comment:
I like the numbering of the unused parameters.

Here's a suggestion: you could lay this out with all the parameters aligned, which would make it even clearer:

~~~ elixir
def won({ x,  x,  x, _4, _5, _6, _7, _8, _9}), do:  x
def won({_1, _2, _3,  x,  x,  x, _7, _8, _9}), do:  x
def won({_1, _2, _3, _4, _5, _6,  x,  x,  x}), do:  x
def won({ x, _2, _3,  x, _5, _6,  x, _8, _9}), do:  x
def won({_1,  x, _3, _4,  x, _6, _7,  x, _9}), do:  x
def won({_1, _2,  x, _4, _5,  x, _7, _8,  x}), do:  x
def won({ x, _2, _3, _4,  x, _6, _7, _8,  x}), do:  x
def won({_1, _2,  x, _4,  x, _6,  x, _8, _9}), do:  x
def won(_), do: false
~~~